practice-two/src/index.html

```diff
@@ -315,14 +315,73 @@ <h3 class="project__item-title">Watch apps</h3>
         </div>
       </section>
       <!-- END TPROJECTS SECTION-->
-
+ 
       <!-- START REVIEW SECTION -->
       <section></section>
       <!-- END REVIEW SECTION-->
 
-      <!-- START PRCING SECTION -->
-      <section></section>
-      <!-- END PRCING SECTION-->
+     <!-- START PRCING SECTION -->
+    <section class="pricing">
+      <div class="container pricing__wrapper">
+        <h2 class="heading-section text-color-secondary pricing__title ">
```

Comment:
I see that you have a class `text-center` in previous merge request. Let use it instead of use `pricing__title`.

Check code style, remove a space here `__title "`